examples/testdata/wasi_args.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
I guess I will make `wasi` subfolder later since a LOT of WASI related test files will be introduced in `examples`.